examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -459,6 +503,38 @@ def _on_error_reported(self, error_report: ErrorReport):
         )
 
 
+@dataclass
+class RearrangeV2AgentConfig:
+    head_sensor_substring: str
+    """Substring used to identify the agent's head sensor."""
+
+    can_change_object_states: bool
+    """Whether the agent can modify object states."""
+
+
+@dataclass
+class RearrangeV2Config:
```

Comment:
These will eventually be changed to structured configs. Not bothering to do this here.